admin/class-create-block-theme-admin.php

```diff
@@ -187,9 +203,31 @@ function clone_theme( $theme, $screenshot ) {
 		header( 'Content-Disposition: attachment; filename=' . $theme['slug'] . '.zip' );
 		header( 'Content-Length: ' . filesize( $filename ) );
 		flush();
-		echo readfile( $filename );
-		die();
+		readfile( $filename );
+		unlink( $filename );
+		exit;
 	}
+
+	function rest_export_theme( $request ) {
+		$theme = $request->get_params();
+		$this->clone_theme( $theme, null );
+	}
+
+	public function register_theme_export() {
+		register_rest_route(
+			'create-block-theme/v1',
+			'/export',
+			array(
+				'methods'             => 'POST',
+				'callback'            => array( $this, 'rest_export_theme' ),
+				'permission_callback' => function () {
+					return true;
+					// return current_user_can( 'edit_theme_options' );
```

Comment:
This keeps returning `401` unauthorized, I don't know why.

Reply:
This is because the ajax request is not authenticated. With the implementation of `apiFetch` this hack is no longer necessary.